### ISSUE TITLE
Distance is rounded beyond 10km

### ISSUE THREAD

**Issue by labiloute** [nicodel/Run-Bike-Hike#110]
For runners (and others), it's important to have an accurate idea of the distance traveled.
The distance is rounded after 10km, which is not really accurate.
See screen shot
Thanks !
![distance_rounded](https://cloud.githubusercontent.com/assets/8760481/8058547/72db887e-0eba-11e5-8bca-0e323b6f89f3.png)

**Comment by nicodel** [comment]
Yes I saw this one on Sunday after a run. Thanks for opening the issue.
